Libplanet.Benchmarks/Libplanet.Benchmarks.csproj

```diff
@@ -0,0 +1,40 @@
+<Project
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
+  Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <RootNamespace>Libplanet.Benchmarks</RootNamespace>
+    <IsPackable>false</IsPackable>
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>7.1</LangVersion>
+    <CodeAnalysisRuleSet>..\Libplanet.Tests.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\Menees.Analyzers.Settings.xml">
+      <Link>Menees.Analyzers.Settings.xml</Link>
+    </AdditionalFiles>
+    <AdditionalFiles Include="..\stylecop.json" />
+  </ItemGroup>
+
+  <PropertyGroup Condition=" '$(MSBuildRuntimeType)'=='Mono' And
+                             '$(BuildingByReSharper)'!='true'">
+    <TargetFramework>net461</TargetFramework>
+  </PropertyGroup>
```

Comment:
As it targets only .NET Core (`<TargetFramework>netcoreapp2.2</TargetFramework>`), isn't this `<PropertyGroup>` unnecessary?

Reply:
I tried to remove this property, but when I remove it, macOS Mono and Unity builds are failed.